apps/website/src/pages/docs/tile.mdx

```diff
@@ -14,14 +14,134 @@ import * as AllExamples from 'examples';
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='core-tile' />
-
 <LiveExample src='Tile.main.tsx'>
   <AllExamples.TileMainExample client:load />
 </LiveExample>
 
 Tiles are containers that display content and actions pertaining to a single item or project. They should be easy to scan for relevant and actionable information. All text should be concise and placed in a way that clearly indicates informational hierarchy.
 
-## Props
+### Composition API
+
+You can build Tiles with subcomponents that are fully customizable.
+
+We provide:
+
+- `Tile.Wrapper`
+- `Tile.ThumbnailArea`
+- `Tile.ThumbnailPicture`
+- `Tile.QuickAction`
+- `Tile.TypeIndicator`
+- `Tile.BadgeContainer`
+- `Tile.IconButton`
+- `Tile.Name`
+- `Tile.NameIcon`
+- `Tile.NameLabel`
+- `Tile.Action`
+- `Tile.ContentArea`
+- `Tile.Description`
+- `Tile.Metadata`
+- `Tile.MoreOptions`
+- `Tile.Buttons`
+
+<LiveExample src='Tile.composition.tsx'>
+  <AllExamples.TileCompositionExample client:load />
+</LiveExample>
+
+### Props API
+
+You can use Tile directly with name, thumbnail, description and other props. This pattern can suffice for simple cases.
+
+A list of all available props can be found [at the end](#props) of this page.
+
+<LiveExample src='Tile.props.tsx'>
+  <AllExamples.TilePropsExample client:load />
+</LiveExample>
+
+## Variants
+
+### Actionable
+
+Click on an actionable tile triggers an action.
+
+Actionable tiles have Tile.Action wrapped around tile name. It adds action to the name, increases click or touch target for said action and keeps accessibility for the best user experience.
+
+<LiveExample src='Tile.actionable.tsx'>
+  <AllExamples.TileActionableExample client:load />
+</LiveExample>
+
+Tile can also link to another page. Add `href` prop to the Tile.Action to make whole tile an anchor link.
```

Comment:
Continuation of https://github.com/iTwin/iTwinUI/pull/1538#discussion_r1311522844

```suggestion
Tile can also link to another page. Add `href` prop to the Tile.Action to make whole tile an anchor link. This is because manually wrapping the Tile with an `<a href="...">` is not recommeneded for accessibility reasons.
```

Reply:
Updated.